src/vinifera_util.cpp

```diff
@@ -227,3 +228,17 @@ bool Vinifera_Generate_Mini_Dump()
     MessageBox(MainWindow, "Failed to create crash dump!\n\n", "Crash dump", MB_OK|MB_ICONASTERISK);
     return false;
 }
+
+
+/**
+ *  Shows a in-game message box.
+ * 
+ *  This has been made its own function because we can not allocate on the stack with
+ *  out patches, so this handles all that within this function scope.
```

Comment:
"with out patches"? Should it be "with our patches" or "without patches" instead?